src/app-error.ts

```diff
@@ -71,6 +86,16 @@ See ${SOURCE_MAP_INFO_URL}`;
       return `Unable to map ${unmappedBytes}/${totalBytes} bytes (${bytesString}%)`;
     }
 
+    case 'InvalidMappingLine': {
+      const { generatedLine, maxLine } = context;
+      return `Your source map refers to generated line ${generatedLine}, but the source only contains ${maxLine} line(s).\nCheck that you are using the correct source map.`;
```

Comment:
Not sure `\n` will work on Unix systems. Please use `os.EOL` or delete `\n`. I think it can be read as one paragraph